tests/adapters/printing.py

```diff
@@ -32,3 +32,17 @@ def pop_and_assert(self, expected_string):
         self._buffer = ""
 
         assert popped_string == modified_expected_string
+
+    def pop_and_assert_lines(self, expected_string):
```

Comment:
Yeah, good idea! 
But I guess as soon as I introduce any kind of ordering, I'll have to adjust the order in the assertions aswell, right? Or what do you mean by "changing the tests"?

Reply:
Oh I see what you mean. Yes I don't mind if the assertions change if the ordering becomes alphabetical.

Reply:
Hm, I'm not able to reproduce the original issue currently. I.e. all tests now pass on Python 3.12 at least.

If I understand correctly, `tox` should run the tests etc. for several python versions.
However, if I try to run `tox` locally, I get the following error:

```
    eps = importlib_metadata.entry_points().get(self.namespace, ())
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'EntryPoints' object has no attribute 'get'
check: exit 1 (0.11 seconds) /home/binz/Repositories/import-linter> flake8 src tests pid=763946
```

This seems to be related to `importlib_metadata` changing stuff, but installing a different version (the proposed solution found by googling) does not work.

How did you setup `tox` locally?

Reply:
While I have still not run tox locally, I fixed the typing error related to `Set[...]` vs `set[...]`. So hopefully, everything passes now

Reply:
Sorry for the delay in replying. If you create a new virtual environment and then pip install tox into it, you should be able to then run it. The only thing is you'll need to make sure you have Python versions accessible for each version of Python you want to run it under. I would suggest just picking one and running that (I use [pyenv](https://github.com/pyenv/pyenv) for that). 

E.g. running tox on 3.10 you can do `tox -epy310`.

Reply:
Ah, I see. I somehow expected `tox` to switch python versions for. No idea why. :D
So, now the output should be stable, since I'm ordering both source and forbidden modules of a forbidden contract.